examples/bench_cudamat.py

```diff
@@ -69,15 +70,15 @@ def main():
     cmt.init()
     cmt.CUDAMatrix.init_random()
     if HEATUP:
-        print "heating up for %g seconds..." % HEATUP,
+        print("heating up for %g seconds..." % HEATUP)
         sys.stdout.flush()
         heatup(HEATUP)
-        print "done."
-    print "small matrix shape:", XS_SHAPE
-    print "large matrix shape:", XL_SHAPE
-    for funcname, func in ifilter(lambda (fn, f): fn.startswith('bench_'),
+        print("done.")
+    print("small matrix shape:", XS_SHAPE)
+    print("large matrix shape:", XL_SHAPE)
+    for funcname, func in filter(lambda f: f[0].startswith('bench_'),
             getmembers(getmodule(main), isfunction)):
-        print "%-15s" % funcname[len('bench_'):],
+        print("%-15s" % funcname[len('bench_'):])
```

Comment:
Hmm, this was meant to be printed without a newline in the end, so we get a tabular output. (Similar for the other print statements that end in a comma.)